pkg/apis/serving/v1alpha1/domainmapping_lifecycle.go

```diff
@@ -35,6 +35,11 @@ func (dm *DomainMapping) GetGroupVersionKind() schema.GroupVersionKind {
 	return SchemeGroupVersion.WithKind("DomainMapping")
 }
 
+// IsReady returns true if the DomainMapping is ready.
+func (dms *DomainMappingStatus) IsReady() bool {
```

Comment:
I think we have moved away from this and are instead providing this method on the type itself, including a Generation check, like:

```golang
// IsReady returns true if the Status condition MetricConditionReady
// is true and the latest spec has been observed.
func (m *Metric) IsReady() bool {
	ms := m.Status
	return ms.ObservedGeneration == m.Generation &&
		ms.GetCondition(MetricConditionReady).IsTrue()
}
```

(this would also make the currently unused ReadyCondition constant used :joy: )

Reply:
ah, I copied this from `Route` but sure that makes sense, lemme update